packages/cli/src/core/getCommands.js

```diff
@@ -3,31 +3,50 @@
  */
 
 import path from 'path';
-import type { CommandT, ProjectCommandT, LocalCommandT } from './types.flow';
 
 import findPlugins from './findPlugins';
 import logger from '../util/logger';
 
+import type { CommandT, ProjectCommandT, LocalCommandT } from './types.flow';
+
+import server from '../server/server';
+import runIOS from '../runIOS/runIOS';
+import runAndroid from '../runAndroid/runAndroid';
+import library from '../library/library';
+import bundle from '../bundle/bundle';
+import ramBundle from '../bundle/ramBundle';
+import eject from '../eject/eject';
+import link from '../link/link';
+import unlink from '../link/unlink';
+import install from '../install/install';
+import uninstall from '../install/uninstall';
+import upgrade from '../upgrade/upgrade';
+import logAndroid from '../logAndroid/logAndroid';
+import logIOS from '../logIOS/logIOS';
+import dependencies from '../dependencies/dependencies';
+import info from '../info/info';
+
 /**
  * List of built-in commands
  */
+
 const loadLocalCommands = (): Array<LocalCommandT> => [
```

Comment:
Doesn't need to be a function anymore. We did it for lazy require call.

Reply:
fixed